test/specs/collections/Form/Field-test.js

```diff
@@ -7,18 +7,16 @@ describe('Field', () => {
   it('has a label', () => {
     render(<Field label='First Name' />)
       .findTag('label')
-      .props.children.should.equal('First Name');
+      .textContent.should.equal('First Name');
   });
 
   it('renders children', () => {
-    render(<Field label='First Name'>yo child</Field>)
-      .findText('yo child');
+    render(<Field>yo child</Field>).assertText('yo child');
```

Comment:
Heads up, `findText` is gone.  Instead, `assertText` exists, you will not get the element back, only a bool.